src/utils/preloader.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+import preloader from '../assets/images/preloader.svg';
+
+const Preloader = () => <img src={preloader} />;
```

Comment:
Consider renaming it to Loader, since "preloader" usually has different function.

Reply:
/fxd